cpp/modmesh/toggle/toggle.cpp

```diff
@@ -34,6 +34,26 @@
 namespace modmesh
 {
 
+int setenv(const char * name, const char * value, int overwrite)
+{
+#ifdef _WIN32
+    int errcode = 0;
+    if (!overwrite)
+    {
+        size_t envsize = 0;
+        errcode = getenv_s(&envsize, NULL, 0, name);
+        if (errcode || envsize)
+        {
+            return errcode;
+        }
+    }
+    return _putenv_s(name, value);
+#else // _WIN32
+    // NOLINTNEXTLINE(concurrency-mt-unsafe) TODO: make the wrapper thread safe.
```

Comment:
Another benefit of the custom wrapper to `setenv()`: we can make it thread safe in the future.